Src/Plugins/Tests.Security/SEC101.SecurePlaintextSecretsTests.cs

```diff
@@ -21,5 +23,11 @@ public SecurePlaintextSecretsTests(ITestOutputHelper outputHelper) : base(output
         [Fact]
         public void SecurePlaintextSecrets_EndToEndFunctionalTests()
             => RunAllTests();
+
+        [Fact]
+        public void AllSecurityRules_ShouldHaveValidators()
```

Comment:
>AllSecurityRules_ShouldHaveValidators [](http://example.com/codeflow?start=20&length=37)

All our end-to-end test files are basically stubs for invoking all tests driven by checked in files, so we should add other tests to it, rather than creating new test source files.